### ISSUE TITLE
Windows/AV

### ISSUE THREAD

**Issue by krakjoe** [krakjoe/pcov#2]
Hi @weltling, it's me again ... are you up for it ? (please)

**Comment by weltling** [comment]
Sure, Joe. I've just added a config, but seems it doesn't trigger AppVeyor. Could you check the settings please?

Btw. `pcov.directory=""` is the most portable solution. Just met that issue as well. There might be still some work needed to support relative paths, when the directory is set.

Thanks.

**Comment by krakjoe** [comment]
I've done the relative cwd support and basic autoselect that will suit most application, and I think it's portable ...

Will check av ...

**Comment by krakjoe** [comment]
Added to AV ...

**Comment by krakjoe** [comment]
We appear to be building, was that automatic ? (yay) 

Thanks again, I appreciate the effort ...

**Comment by weltling** [comment]
Yeah, looks good with AppVeyor :) I'll check for PECL once it's that far. Closing.

Thanks.

**Comment by krakjoe** [comment]
Thanks very much :)
